docs/Migrating-v4.md

```diff
@@ -205,8 +209,9 @@ EX: `{% include '@ecl/form-group/form-group.html.twig' with {
 
 #### Description list
 
-- Lists with links, regardless of the type, always expect the `ecl-link--standalone` class to prevent the underlining.
+- Lists with links, in most cases, expect the `ecl-link--standalone` class to prevent the underlining. The only excpetion is the taxonomy list.
```

Comment:
```suggestion
- Lists with links, in most cases, expect the `ecl-link--standalone` class to prevent the underlining. The only exception is the taxonomy list.
```